### DISCUSSION TITLE
Show current speed of interface

### DISCUSSION THREAD

**Discussion by elmorekevin** [snappysnappydog/Simple-IP-Config#111] [Ideas]
The current negotiated speed of the interface can be found using this windows command:
wmic NIC where NetEnabled=true get Name, Speed

Could Simple IP Config also show this interface speed where it shows the Adapter State? Could the Duplex value be added as well?

Thanks,
Kevin

**Comment by KurtisLiggett** [comment]
I'll look into this.

**Comment by zapy85** [comment]
was just about to make a new request for this as well, would be nice if it could show the linked speed and duplex of said adapter (at least the wired ones)

@KurtisLiggett are you on irc, discord or similar and want help in investigating these things and helping out?
